blog/2024-01-30-FAQ.md

```diff
@@ -95,6 +95,23 @@ You can close your position, but your channel will remain open until you decide
 
 Depending on the size of this channel, you'll be able to open larger or smaller positions in the trading section, or hold more or less synthetic currency in your wallet (to be released)
 
+### Why my position has an expiry?
+
+This is how futures contracts work, and so are discreet log contracts.
+
+A position has an expiry of 1 week.
+
+You, as the trader can decide to close your position anytime by clicking _Close position_
+
+### What if I don't want my position to expire after 1 week?
+
+A position has an expiry of 1 week, i.e. it always expires on Sunday, 15:00 CET.
+
+We introduced a quick and simple way to renew it, codenamed _rollover weekends_ :
+If you come online between Friday, 15:00 CET and Sunday, 15:00 CET your position extends for another week.
+
+If you've accidentally rolled over a position you didn't want to keep open, you can decide to close your position at any time by clicking on _Close position_.
```

Comment:
I guess we can mention that atm there is no downside to rolling over your position, but that eventually we will introduce a funding rate.